src/main/java/com/agency04/sbss/pizza/controller/CustomerController.java

```diff
@@ -0,0 +1,54 @@
+package com.agency04.sbss.pizza.controller;
+
+import com.agency04.sbss.pizza.model.Customer;
+import com.agency04.sbss.pizza.repo.Repository;
+import com.agency04.sbss.pizza.service.PizzaDeliveryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+public class CustomerController {
+
+    @Autowired
+    private PizzaDeliveryService pizzaDeliveryService;
+
+    @GetMapping("/customer/{username}")
+    public Customer getCustomer(@PathVariable String username) {
+
+        Customer customer =  Repository.getInstance().getCustomerByUsername(username);
```

Comment:
napravi `customerService` koji ce zvati `customerRepository` i onda to dodas ovdje.

Reply:
`customerRepository` u week-3?

Reply:
`Repository.getInstance(). 
`
mislio sam na ovo.

Reply:
ali svejedno ti treba servis koji ce odradivati ovu logiku.

previse je logike da bi bila u controlleru.